library/src/main/java/com/q42/q42stats/library/Q42StatsConfig.kt

```diff
@@ -0,0 +1,9 @@
+package com.q42.q42stats.library
+
+data class Q42StatsConfig(
+    val fireBaseProject: String,
+    val firebaseCollection: String,
+    /** Data collection is skipped when less than this many seconds have passed
+     * since the previous run */
+    val minimumSubmitInterval: Long
```

Comment:
The name `minimumSubmitIntervalMs` would be clearer, but using a Duration is even more clear.

Reply:
goiing with minimumSubmitIntervalMs, as the time lib is experimental and [explicitly discouraged for libraries](https://kotlinlang.org/api/latest/jvm/stdlib/kotlin.time/-experimental-time/)